src/node_report.cc

```diff
@@ -878,16 +878,12 @@ void PrintNativeStack(std::ostream& out) {
       DWORD dwOffset = 0;
       IMAGEHLP_LINE64 line;
       line.SizeOfStruct = sizeof(line);
+      snprintf(buf, sizeof(buf), "%2d: [pc=0x%p]", i, reinterpret_cast<void*>(pSymbol->Address));
+      out << buf << " " << pSymbol->Name << " [+";
```

Comment:
I wonder if it's worth inserting a comment with sample output lines (one for each branch of the nested `if` statement) as it's now harder to tell what the output is supposed to look like as the formatting is split between the `snprintf` prior to the if statement and `<<` statements both here and in the nested `if` statement.